http-gateways/PATH_GATEWAY.md

```diff
@@ -589,10 +593,6 @@ Data sent with HTTP response depends on the type of requested IPFS resource:
   - Opaque bytes, see [application/vnd.ipld.raw](https://www.iana.org/assignments/media-types/application/vnd.ipld.raw)
 - CAR
   - CAR file or stream, see [application/vnd.ipld.car](https://www.iana.org/assignments/media-types/application/vnd.ipld.car)
-<!-- TODO: https://github.com/ipfs/go-ipfs/issues/8823
-- dag-json / dag-cbor
-  - See [https://github.com/ipfs/go-ipfs/issues/8823](https://github.com/ipfs/go-ipfs/issues/8823)
--->
```

Comment:
We may want to  update this to cover JSON/CBOR/DAG-JSON/DAG-CBOR  like we did for Block and CAR above.

Reply:
Updated, also added in the TAR IPIP PR.